conf.d/health_alarm_notify.conf

```diff
@@ -181,6 +182,14 @@ DEFAULT_RECIPIENT_EMAIL="root"
 
 
 #------------------------------------------------------------------------------
+# Alexa
+# See http://www.thomptronics.com/notify-me
+# multiple recipients can be given like this:
+#                  "USERTOKEN1 USERTOKEN2 ..."
```

Comment:
There is no notion of variable `USERTOKEN` anywhere in code regarding "notify me". Also this should be an example of full variable assignment.